### DISCUSSION TITLE
cartoon network [it]

### DISCUSSION THREAD

**Discussion by dantheman121** [iptv-org/community#892] [Q&A]
### Channel ID

CartoonNetwork.it

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
According to our database this channel was closed on 15 December 2020.

https://iptv-org.github.io/channels/it/CartoonNetwork
